instalooter/pages.py

```diff
@@ -72,20 +75,28 @@ def _page_loader(self, session, rhx):
                     self._last_page = data = res.json()
                 # Yield that same data until cursor is updated
                 while self._cursor == cursor:
-                    yield data['data']
+                    if 'data' in data:
+                        retry_count = 0
+                        yield data['data']
+                    else:
+                        last_exception = RuntimeError("Query rate exceeded (wait before next run)")
+                        retry_count += 1
+                        if data.get('message') == 'rate limited':
+                            raise last_exception
```

Comment:
here we want to always raise: `RuntimeError("Query rate exceeded (wait before next run)")`